services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerClusterConfig.java

```diff
@@ -654,7 +654,9 @@ public VeniceControllerClusterConfig(VeniceProperties props) {
     this.adminCheckReadMethodForKafka = props.getBoolean(ADMIN_CHECK_READ_METHOD_FOR_KAFKA, true);
     this.controllerClusterName = props.getString(CONTROLLER_CLUSTER, "venice-controllers");
     this.controllerResourceInstanceGroupTag = props.getString(CONTROLLER_RESOURCE_INSTANCE_GROUP_TAG, "");
-    this.controllerInstanceTagList = props.getList(CONTROLLER_INSTANCE_TAG_LIST, Collections.emptyList());
+    this.controllerInstanceTagList = props.getString(CONTROLLER_INSTANCE_TAG_LIST, "").isEmpty()
+        ? Collections.emptyList()
+        : props.getList(CONTROLLER_INSTANCE_TAG_LIST, Collections.emptyList());
```

Comment:
This is a bit hard to read. Can we split it in two lines? One to read the string, other to make it a list?

Also, since we're just using the string to do the emptiness check, let's keep null as the default and do a null check instead